### DISCUSSION TITLE
I am getting this error while building new version of this code.. Kindly help!!

### DISCUSSION THREAD

**Discussion by shary998** [florisboard/florisboard#1802] [General]
Build command failed.
Error while executing process C:\Users\Shaheryar\AppData\Local\Android\Sdk\cmake\3.18.1\bin\ninja.exe with arguments {-C D:\Floris New Board\florisboard-master\app\.cxx\Debug\6621703z\arm64-v8a florisboard-native}
ninja: Entering directory `D:\Floris New Board\florisboard-master\app\.cxx\Debug\6621703z\arm64-v8a'

ninja: error: 'D:/Floris New Board/florisboard-master/app/src/main/icu4c/prebuilt/jniLibs/arm64-v8a/libicuuc.a', needed by 'D:/Floris New Board/florisboard-master/app/build/intermediates/cxx/Debug/6621703z/obj/arm64-v8a/libflorisboard-native.so', missing and no known rule to make it

**Comment by patrickgold** [comment]
You need to initialize the git submodules, which contains files required to build this project: 

``git submodule update --init --recursive``

Then it should work.
